models/quality_measures/intermediate/nqf0041_influenza_immunization/quality_measures__int_nqf0041_numerator.sql

```diff
@@ -0,0 +1,148 @@
+{{ config(
+     enabled = var('quality_measures_enabled',var('claims_enabled',var('clinical_enabled',var('tuva_marts_enabled',False))))
+    | as_bool
+   )
+}}
+
+with denominator as (
+
+    select 
+          patient_id
+        , performance_period_begin
+        , performance_period_end
+        , measure_id
+        , measure_name
+        , measure_version
+    from {{ ref('quality_measures__int_nqf0041_denominator') }}
+
+)
+
+, influenza_vaccination_code as (
+
+    select
+          code
+        , code_system
+        , concept_name
+    from {{ ref('quality_measures__value_sets') }}
+    where lower(concept_name) in (
+          'influenza vaccination'
+        , 'influenza vaccine'
+        , 'influenza virus laiv immunization'
+        , 'influenza virus laiv procedure'
+    )
+
+)
+
+, procedure_vaccination as (
+
+    select
+        patient_id
+      , procedure_date
+      , source_code_type
+      , source_code
+    from {{ ref('quality_measures__stg_core__procedure') }} as procedures
+    inner join influenza_vaccination_code
+        on procedures.source_code = influenza_vaccination_code.code
+        and procedures.source_code_type = influenza_vaccination_code.code_system
+
+)
+
+, claims_vaccination as (
+    
+    select 
+          patient_id
+        , coalesce(claim_start_date,claim_end_date) as min_date
+        , coalesce(claim_end_date,claim_start_date) as max_date
+    from {{ ref('quality_measures__stg_medical_claim') }} medical_claim
+    inner join influenza_vaccination_code
+        on medical_claim.hcpcs_code = influenza_vaccination_code.code
+
+)
+
+, qualifying_procedures as (
+
+    select
+          procedure_vaccination.patient_id
+        , procedure_vaccination.procedure_date as evidence_date
+    from procedure_vaccination
+    inner join {{ ref('quality_measures__int_nqf0041__performance_period') }} pp
+        on procedure_date between 
+            pp.lookback_period_august and
+                pp.performance_period_end
```

Comment:
if i am not wrong, this will be excluding procedures happening at first quarter.
is this something we are trying to do in this PR?

Reply:
Lookback period august is dynamically set to last year's august if measurement period is the first quarter of this year. So, will not be excluding procedure.